guides/doc-Release_Notes/master.adoc

```diff
@@ -14,7 +14,11 @@ endif::[]
 
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.7.0.adoc[leveloffset=+1]
```

Comment:
I'd list Foreman last. Foreman 3.7.0 was released before Katello 4.9.0.

My reasoning is that you remember I updated to X.Y.Z last, so I read till there. That probably needs to written down somewhere.

Reply:
I'll look at updating the instructions from the procedure. Thanks!